msipackage/package.wix.in

```diff
@@ -350,7 +350,10 @@
                         </RegistryKey>
                     </RegistryKey>
                 </Component>
-                <Files Include="*.dll"/>
+                <File Id="wslsettings.dll" Source="${PACKAGE_INPUT_DIR}/wslsettings/wslsettings.dll"/>
```

Comment:
What's the reason for that change ?

Reply:
We want to make sure we are copying the signed version of wslsettings.dll, the one in bin might have been overwritten.